soroban-sdk/src/logs.rs

```diff
@@ -62,21 +62,21 @@ use crate::{env::internal::EnvBase, Env, Val};
 /// let value = 5;
 /// log!(&env, "a log entry", value, Symbol::short("another"));
 ///
-/// use soroban_sdk::testutils::Logger;
-/// let logentry = env.logger().all().last().unwrap().clone();
+/// use soroban_sdk::testutils::Logs;
+/// let logentry = env.logs().all().last().unwrap().clone();
 /// assert!(logentry.contains("[\"a log entry\", 5, another]"));
 /// # }
 /// ```
 #[macro_export]
 macro_rules! log {
     ($env:expr, $fmt:literal $(,)?) => {
         if cfg!(debug_assertions) {
-            $env.logger().log($fmt, &[]);
+            $env.logs().log($fmt, &[]);
         }
     };
     ($env:expr, $fmt:literal, $($args:expr),* $(,)?) => {
         if cfg!(debug_assertions) {
-            $env.logger().log($fmt, &[
+            $env.logs().log($fmt, &[
```

Comment:
I'm not a huge fan of `logs().log` honestly; either `logger().log()` or `logs().add()` looks more natural to me (i.e. either 'actor.action' or 'data_structure.mutator`). Though I suppose this is rarely used directly, so I don't fee too strongly. WDYT?

Reply:
Maybe `logs().write()`? Otherwise .add sgtm.

Reply:
Actually, add is better, as the interface makes it more like it's a collection.